fleet_adapter_mir_tasks/fleet_adapter_mir_tasks/dispatch_pickup.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
```

Comment:
I'm worried that calling this file `dispatch_pickup.py` might lead users to think it only does a pickup and not a drop-off, when in reality it does an entire delivery. I assume you were trying to avoid using the same name as the `dispatch_delivery.py` file in `rmf_demos_tasks`, but we shouldn't use a different name if it comes at the cost of clarity.

This file will be namespaced inside of `fleet_adapter_mir_tasks` so there won't be a conflict if it has the same name as `dispatch_delivery` from `rmf_demos_tasks`. Users will run it like this:
```
$ ros2 run fleet_adapter_mir_tasks dispatch_delivery ...
```